robottelo/libvirt_discovery.py

```diff
@@ -56,7 +56,13 @@ def __init__(
         else:
             self.image_dir = image_dir
         if mac is None:
-            self.mac = gen_mac(multicast=False, locally=True)
+            # fe:* MAC range is considered reserved in libvirt
+            while True:
+                mac = gen_mac(multicast=False, locally=True)
+                if not mac.startswith(u'fe'):
```

Comment:
OK, I just read the related issue, we cannot generate macs starting with `fe` so maybe the better is to address that problem in `fauxfactory`?